qaframework-bdd-tests/src/test/resources/org/openmrs/uitestframework/test-local.properties

```diff
@@ -1,4 +1,4 @@
-webapp.url=http://localhost:8080/openmrs
+webapp.url=https://qa-refapp.openmrs.org/openmrs
```

Comment:
why are we changing this?

Reply:
@mherman22 there's something @sherrif10  and I were testing